.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+	"env": {
+		"browser": true,
+		"node": true,
+		"jest": true,
+		"es2021": true
+	},
+	"parserOptions": {
+		"ecmaVersion": 13,
+		"sourceType": "script"
+	},
+	"extends": ["eslint:recommended", "plugin:jsdoc/recommended", "prettier"],
+	"plugins": ["jsdoc", "html"],
+	"rules": {
+		"no-unused-vars": ["error", {"args": "none"}]
+	},
+	"overrides": [
+		{
+			"files": ["src/scripts/bttvIntegration.js", "src/scripts/handleChat.js"],
```

Comment:
Can you help me understand the choice to override these two scripts?

Reply:
The project mixes uses of CommonJS and ESM.
I marked the files that use ESM as using ESM, all other files are considered CommonJS by ESLint as that's the default.
No idea what the correct way to go is here. The commonJS and ESM mix is a mess in the JS ecosystem.

Reply:
Ah, good deal - thanks for the explanation!